lib/open_feature/evaluation_options.rb

```diff
@@ -5,5 +5,6 @@ module OpenFeature
   # Additional options to be supplied to client during flag evaluation.
   class EvaluationOptions < T::Struct
     const :hooks, T::Array[Hook], default: []
+    const :hints, T::Hash[String, T.untyped], default: {}
```

Comment:
probably don't need this yet, as it is getting dropped. 

also this is not in the specification I believe

but I saw that the Java SDK added it here. 

Besides that, I still don't have a clear idea about what hints are used for. 😆

Reply:
Yea, let's drop hints for right now and not deal with it in this PR 👍🏻